page/press-release.php

```diff
@@ -0,0 +1,109 @@
+<link rel="stylesheet" href="css/template-1.css">
+<div class="full text-center">
+    <span class="garis"></span>
+    <h1 class="display-5 text-light"><dt><b>SIARAN PERS</b></dt></h1>
+</div>
+
+<div class="container mt-5">
+    <div class="row">
+        <div class="col-md-8">
+
+            <?php
+                $select = mysqli_query($connection, "select * from berita where kategori='press-release'");
+                while($data = mysqli_fetch_array($select)){
+                    if($data[4]==null || $data[4]==""){
+                        $thumbnail = "#";
+                    }else{
+                        $thumbnail = "news/".explode("-", $data[4])[0]."/gambar/".explode("-", $data[4])[1];
+                    }
+                    ?>
+
+                    <div class="row article-berita mb-5 pb-5">
+                        <div class="col-md-2 text-center">
+                            <div class="tanggal-group float-right">
+                                <div class="hari"><?php echo date("d", strtotime($data[2])) ?></div>
+                                <div class="tanggal"><?php echo date("M", strtotime($data[2])) ?></div>
+                                <div class="tanggal"><?php echo date("Y", strtotime($data[2])) ?></div>
+                            </div>
+                        </div>
+                        <div class="col-md-4">
+                            <h5 class="font-weight-bolder mb-4">
+                                <a href="index.php?page=view-news&id=<?php echo $data[0] ?>"><?php echo $data[1] ?></a>
+                            </h5>
+                            <div class="deskripsi">
+                                <div class="hide">
+                                    <?php include(str_replace("../../", "", $data[3])) ?>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="col-md-6">
+                            <a href="index.php?page=view-news&id=<?php echo $data[0] ?>"><img src="<?php echo $thumbnail ?>" class="img-fluid" alt=""></a>
```

Comment:
Codacy found an issue: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$data[0]'.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)